pkg/model/manifest_test.go

```diff
@@ -141,29 +141,43 @@ var equalitytests = []struct {
 	},
 	{
 		"ImageTarget.ConfigurationRef unequal",
-		Manifest{}.WithImageTarget(ImageTarget{ConfigurationRef: img1}),
-		Manifest{}.WithImageTarget(ImageTarget{ConfigurationRef: img2}),
+		Manifest{}.WithImageTarget(ImageTarget{Refs: container.RefSet{ConfigurationRef: img1}}),
+		Manifest{}.WithImageTarget(ImageTarget{Refs: container.RefSet{ConfigurationRef: img2}}),
 		false,
 		true,
 	},
 	{
 		"ImageTarget.ConfigurationRef equal",
-		Manifest{}.WithImageTarget(ImageTarget{ConfigurationRef: img1}),
-		Manifest{}.WithImageTarget(ImageTarget{ConfigurationRef: img1}),
+		Manifest{}.WithImageTarget(ImageTarget{Refs: container.RefSet{ConfigurationRef: img1}}),
+		Manifest{}.WithImageTarget(ImageTarget{Refs: container.RefSet{ConfigurationRef: img1}}),
 		true,
 		false,
 	},
 	{
-		"ImageTarget.DeploymentRef unequal",
-		Manifest{}.WithImageTarget(ImageTarget{DeploymentRef: img1.AsNamedOnly()}),
-		Manifest{}.WithImageTarget(ImageTarget{DeploymentRef: img2.AsNamedOnly()}),
+		"ImageTarget.LocalRef unequal",
+		Manifest{}.WithImageTarget(ImageTarget{Refs: container.RefSet{LocalRef: img1.AsNamedOnly()}}),
+		Manifest{}.WithImageTarget(ImageTarget{Refs: container.RefSet{LocalRef: img2.AsNamedOnly()}}),
 		false,
 		true,
 	},
 	{
-		"ImageTarget.DeploymentRef equal",
-		Manifest{}.WithImageTarget(ImageTarget{DeploymentRef: img1.AsNamedOnly()}),
-		Manifest{}.WithImageTarget(ImageTarget{DeploymentRef: img1.AsNamedOnly()}),
+		"ImageTarget.LocalRef equal",
```

Comment:
fwiw, it doesn't seem like adding tests here is adding a ton of value. Now that we've added the comparator library, they're basically just testing the comparator library.

Reply:
the what now?